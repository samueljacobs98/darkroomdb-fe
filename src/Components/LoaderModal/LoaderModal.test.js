import { render } from "@testing-library/react";
import LoaderModal from "./LoaderModal";

it("should match component snapshot", () => {
  const { container } = render(<LoaderModal />);

  expect(container).toMatchSnapshot();
});
