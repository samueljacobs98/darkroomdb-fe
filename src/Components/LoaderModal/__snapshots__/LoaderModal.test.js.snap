// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match component snapshot 1`] = `
<div>
  <div
    class="loader-modal"
  >
    <img
      alt=""
      class="loader-modal__loader"
      src="loader.gif"
    />
  </div>
</div>
`;
